ethportal-api/src/types/jsonrpc/endpoints.rs

```diff
@@ -65,7 +65,7 @@ pub enum HistoryEndpoint {
 }
 
 /// Beacon network JSON-RPC endpoints. Start with "portal_beacon" prefix
-#[derive(Debug, PartialEq, Eq, Clone)]
```

Comment:
Yeah, we are not using it and it adds more complexity because all of the inner types must implement `Eq`  which is more restrictive and requires some additional tweaks.